### PR TITLE
Fix bug Home components doesn't have id to navigate to #37

### PR DESCRIPTION
**Pull request by thuyvy3110**
Added id for Home components

### CODE REVIEW

File: views/components/Home.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import * as React from 'react'
-
+import { HOME } from '~/consts/pages'
 import styled from 'styled-components/macro'
 
 const Home = () => (
-  <HomeSection>
+  <HomeSection id={HOME} className="container">
```
Comment (DeKal) [inline views/components/Home.js]: remove className="container

Comment (DeKal) [reply views/components/Home.js]: also update the snapshot again :<

Comment (thuyvy3110) [reply views/components/Home.js]: Removed className, updated snapshot

Comment (thuyvy3110) [reply views/components/Home.js]: > remove className="container

Removed

Comment (thuyvy3110) [reply views/components/Home.js]: > also update the snapshot again :<

Updated

**Pull request merged**